src/class/bth/bth_device.c

```diff
@@ -214,7 +214,7 @@ bool btd_control_xfer_cb(uint8_t rhport, uint8_t stage, tusb_control_request_t c
     }
     else return false;
 
-    return tud_control_xfer(rhport, request, &_btd_itf.hci_cmd, request->wLength);
+    return tud_control_xfer(rhport, request, &_btd_itf.hci_cmd, sizeof(_btd_itf.hci_cmd));
```

Comment:
I would also consider adding similar size check in call to `tud_bt_hci_cmd_cb()` in line 224, as not to deceive BT stack about size of command buffer if wLength can not be trusted.
```c
if (tud_bt_hci_cmd_cb) tud_bt_hci_cmd_cb(&_btd_itf.hci_cmd,
              tu_min16(sizeof(_btd_itf.hci_cmd), request->wLength));
```